jobs/JWAVE_GEMPAK

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+# March-2020 RPadilla & JHAlves                                       +
+#           - For GEFSv12.0                                           +
+#                                                                     +
+######################################################################
+set -xa
+# #### 08/25/1999 ###################
+# SET SHELL PROCESSING VARIABLES
+# ###################################
+export PS4='$SECONDS + ' 
+date
+# 
+#####################################
+# SET SOME VARIABLES
+export grids=${grids:-'glo_30m at_10m ep_10m wc_10m ao_9km'}
+export NET=${NET:-gfs}
+export RUN=${RUN:-gfs}
+export COMPONENT=${COMPONENT:-wave}
+export RUNwave=${RUNwave:-${RUN}${COMPONENT}}
+#export modID=${modID:-${RUN}${NET}}
+export fstart=${fstart:-0}
+export FHMAX_WAV=${FHMAX_WAV:-180}  #180 Total of hours to process
```

Comment:
This is probably too many settings for a j-job. NCO wants j-jobs to be very lean. These settings should be pushed down or moved to a config/parm file.